main.py

```diff
@@ -22,3 +20,26 @@
         logger=config.get("logger", "tensorboard"),
     )
     trainer.train()
+
+
+if __name__ == "__main__":
+    parser = flags.get_parser()
+    args = parser.parse_args()
+    config = build_config(args)
+
+    if args.submit:  # Run on cluster
+        executor = submitit.AutoExecutor(folder=args.logdir)
+        executor.update_parameters(
+            name=args.identifier,
+            mem_gb=args.slurm_mem,
+            timeout_min=args.slurm_timeout * 60,
+            slurm_partition=args.slurm_partition,
+            gpus_per_node=args.num_gpus,
+            cpus_per_task=(args.num_workers + 1),
+            ntasks_per_node=1,
```

Comment:
Should we make this `tasks_per_node`? `ntasks_per_node` seems to be deprecated for the slurm executor

Reply:
Done.